lua/default-config.lua

```diff
@@ -149,6 +149,13 @@ O = {
   },
 }
 
+local function configure_plugin(plugin_name)
+  status_ok, plugin = pcall("require", plugin_name)
+  if status_ok then
+    plugin.config()
+  end
+end
+
```

Comment:
Could you move this function to `lv-utils`?

Reply:
Sure